libs/blocks/gnav/gnav-contextual-search.js

```diff
@@ -0,0 +1,97 @@
+import { fetchBlogArticleIndex } from '../article-feed/article-feed.js';
+import { getArticleTaxonomy, buildArticleCard } from '../article-feed/article-helpers.js';
```

Comment:
The mark's yellow color is quite strong in my opinion. Do we want to style it so it looks more like how it is on adobe.com? 
![image](https://user-images.githubusercontent.com/115231412/224002285-4eb83dfa-51c3-473d-99b3-5f7559ddaf03.png)

Reply:
Could you attach a picture as a reference? On adobe.com,  I could only find the text bold as a highlight.

Reply:
@saugatmalla on blog.adobe.com you can see a less bright yellow color.

I just checked your css. You have the correct `<mark>` color added (`background-color: rgba(255, 208, 0, 0.3);`), but the browser default is taking over and showing the BRIGHT yellow. You'll need to add the css outlined in the screenshot, or something similar, to correct for that. This will allow the custom `<mark>` color to show.
![image](https://user-images.githubusercontent.com/2539954/224177373-6971b94a-6f3c-4b1f-a446-0500a6e70e62.png)